CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ APPLY_DEFAULT_APPLE_CONFIGURATION()
 # Add the different required and optional dependencies
 ADD_REQUIRED_DEPENDENCY("eigen3 >= 3.0.5")
 ADD_REQUIRED_DEPENDENCY("eigenpy")
-ADD_REQUIRED_DEPENDENCY("pinocchio >= 2.1.9")
+ADD_REQUIRED_DEPENDENCY("pinocchio >= 2.2.1")
```

Comment:
Which feature required this version bump?

Reply:
Efficient computation of centroidal momentum. Previous Pinocchio method constrained us to call twice forwardKinematics

Reply:
Ok, thank you.